de.dlr.sc.virsat.model.concept.test/resources/expectedOutputFilesForGenerators/StructuralElementInstanceValidatorTest.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package testConcept.validator;
+
+// *****************************************************************
+// * Import Statements
+// *****************************************************************
+
+
+
+// *****************************************************************
+// * Class Declaration
+// *****************************************************************
+
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Auto Generated Class inheriting from Generator Gap Class
+ * 
+ * This class is generated once, do your changes here
+ * 
+ * 
+ *
+ */
+	public class StructuralElementInstanceValidatorTest extends AStructuralElementInstanceValidatorTest {
+
+	@Test	
+	public void testStructuralElementInstanceValidator() { 
+	StructuralElementInstanceValidator validator = new StructuralElementInstanceValidator();

```

Comment:
Could we improve the indentation here? Content of a method should be indented ;)